src/cli_plugin/install/settings.js

```diff
@@ -0,0 +1,48 @@
+import expiry from 'expiry-js';
+import { intersection } from 'lodash';
+import { resolve } from 'path';
+import { arch, platform } from 'os';
+
+function generateUrls(settings) {
+  const { version, plugin } = settings;
```

Comment:
If you don't need access to `settings` itself, you can just do this in the function declaration:

```
function generateUrls({ version, plugin }) {
```